coralproject/cay
# Websocket server

## What does this PR do?

Integrate socket.io with our redux store using a middleware.
## How do I test this PR?

From the client to the server:
- Dispatch an action including a `publish` property with value `true` at the body
- It should be recognized by socket.io (to test it go to the server and add this code inside the io.on('connection') handler: `socket.on('action', function(payload){console.log(payload)})`)

From the server to the client:
- Go to the dev-server file and add this code inside the `io.on('connection')` handler: `io.emit('action', {type: 'A_TYPE_THAT_TRIGGERS_SOMETHING_IN_CAY'})`
- See that it works :D

@coralproject/frontend
